newsfeeds/tasks.py

```diff
@@ -30,7 +33,7 @@ def fanout_newsfeeds_main_task(tweet_id, tweet_user_id):
         fanout_newsfeeds_batch_task.delay(tweet_id, follower_ids[batch_from:batch_end])
         batch_from, batch_end = batch_end, batch_end + FANOUT_BATCH_SIZE
 
-    return f'{len(follower_ids) // FANOUT_BATCH_SIZE + 1} batches created, '\
+    return f'{math.ceil(len(follower_ids) / FANOUT_BATCH_SIZE)} batches created, '\
```

Comment:
It also can be written as the following without additional dependency `math.ceil()`
```python
    return f'{(len(follower_ids) - 1) // FANOUT_BATCH_SIZE + 1} batches created, '\
```